From: hololeap <hololeap@protonmail.com>
Signed-off-by: hololeap <hololeap@protonmail.com>
Bug: https://github.com/dhall-lang/dhall-haskell/issues/2323

These doctests depend on the ordering of a HashMap being the same consistantly.
Disable them.

diff -urN dhall-toml-1.0.0/src/Dhall/DhallToToml.hs dhall-toml-1.0.0-r1/src/Dhall/DhallToToml.hs
--- dhall-toml-1.0.0/src/Dhall/DhallToToml.hs	2001-09-08 19:46:40.000000000 -0600
+++ dhall-toml-1.0.0-r1/src/Dhall/DhallToToml.hs	2021-10-31 15:48:13.588718627 -0600
@@ -215,10 +215,11 @@
 >>> import Toml.Type.Printer
 >>> f = makeRecordField
 >>> let toml = dhallToToml $ RecordLit [("foo", f $ NaturalLit 1), ("bar", f $ TextLit "ABC")]
->>> toml
-Right (TOML {tomlPairs = fromList [("foo" :| [],Integer 1),("bar" :| [],Text "ABC")], tomlTables = fromList [], tomlTableArrays = fromList []})
->>> fmap Toml.Type.Printer.pretty toml
-Right "bar = \"ABC\"\nfoo = 1\n"
+
+> >>> toml
+> Right (TOML {tomlPairs = fromList [("foo" :| [],Integer 1),("bar" :| [],Text "ABC")], tomlTables = fromList [], tomlTableArrays = fromList []})
+> >>> fmap Toml.Type.Printer.pretty toml
+> Right "bar = \"ABC\"\nfoo = 1\n"
 -}
 dhallToToml :: Expr s Void -> Either CompileError TOML
 dhallToToml e0 = do
